package.json

```diff
@@ -22,7 +22,7 @@
     "clean": "rm -rf dist/ deploy/docs",
     "lint": "eslint --ext .ts,.tsx.js,.jsx .",
     "start": "yarn docs:start",
-    "test": "jest"
+    "test": "TZ=UTC jest"
```

Comment:
This ensures that dates always get created in the UTC timezone in tests, so that we have consistency b/w GitHub actions + local.

Reply:
TIL nice 👍